spec.html

```diff
@@ -37,6 +37,143 @@ <h1>Introduction</h1>
+        <emu-alg>
+          1. If NewTarget is *undefined*, let _newTarget_ be the active function object, else let _newTarget_ be NewTarget.
+          1. Let O be ? OrdinaryCreateFromConstructor(_newTarget_, `"%AggregateError.prototype%"`, « [[ErrorData]] »).
+          1. Let _errorsArray_ be ! CreateArrayFromList(? IterableToList(_errors_)).
+          1. Perform ! CreateMethodProperty(_O_, `"errors"`, _errorsArray_).
+          1. If _message_ is not _undefined_, then
+            1. Let msg be ? ToString(_message_).
+            1. Perform ! CreateMethodProperty(_O_, `"message"`, _msg_).
+          1. Return _O_.
+        </emu-alg>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-the-aggregateerror-constructors">
+      <h1>Properties of the _AggregateError_ Constructor</h1>
+      <p>The _AggregateError_ constructor:</p>
+      <ul>
+        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error%.</li>
+        <li>has a `name` property whose value is the String value `"<var>AggregateError</var>"`.</li>
+        <li>has the following properties:</li>
+      </ul>
+
+      <emu-clause id="sec-aggregate-error.prototype">
+        <h1>AggregateError.prototype</h1>
+        <p>The initial value of <code><var>AggregateError</var>.prototype</code> is a _AggregateError_ prototype object (<emu-xref href="#sec-properties-of-the-aggregateerror-prototype-objects"></emu-xref>).</p>
+        <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+      </emu-clause>
+    </emu-clause>
+
+    <emu-clause id="sec-properties-of-the-aggregateerror-prototype-objects">
+      <h1>Properties of the _AggregateError_ Prototype Object</h1>
+      <p>The _AggregateError_ prototype object:</p>
+      <ul>
+        <li>is an ordinary object.</li>
+        <li>is not an Error instance and does not have an [[ErrorData]] internal slot.</li>
+        <li>has a [[Prototype]] internal slot whose value is the intrinsic object %Error.prototype%.</li>
+      </ul>
+
+      <emu-clause id="sec-aggregate-error.prototype.constructor">
+        <h1>_AggregateError_.prototype.constructor</h1>
+        <p>The initial value of the `AggregateError.prototype.constructor` is the intrinsic object %_AggregateError_% (<emu-xref href="#sec-aggregate-error-constructors"></emu-xref>).</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregate-error.prototype.name">
+        <h1>_AggregateError_.prototype.name</h1>
+        <p>The initial value of the `AggregateError.prototypename` is the String value `AggregateError`.</p>
+      </emu-clause>
+
+      <emu-clause id="sec-aggregate-error.prototype.errors">
+        <h1>_AggregateError_.prototype.errors</h1>
+        <p>The initial value of the `AggregateError.prototypeerrors` is an empty Array.</p>
```

Comment:
```suggestion
        <p>The initial value of the `AggregateError.prototype.errors` is an empty Array.</p>
```

however, this shouldn’t be here - it either needs to be deeply frozen, or an accessor that always returns a new array, or absent entirely. See #33

Reply:
just added this by hand because of 
<img width="248" alt="image" src="https://user-images.githubusercontent.com/1507086/65614044-9dc21280-dfbf-11e9-94dc-e55e64712e75.png">

Reply:
again tho this shouldn’t exist on the prototype as a mutable object in a data property.